dnf/module/module_base.py

```diff
@@ -437,7 +437,7 @@ def _profile_report_formater(self, modulePackage, default_profiles, enabled_str)
                 else ", "
         return profiles_str[:-2]
 
-    def _module_strs_formater(self, modulePackage, markActive=False):
+    def _module_strs_formatter(self, modulePackage, markActive=False):
```

Comment:
Even though marked as private by starting with an underscore, methods in Python are never private and it is possible someone is actually using them. @m-blaha should know better whether we should be fixing these.

Reply:
We do change these "private" methods when needed. I do not object to this fix.

Reply:
Yep, I've noticed that methods starting with `_` seem only to be used internally and don't show up in any documentation/API reference.

A lot of languages don't have means of protecting data or functions, but rely on common sense and developer civility to not do actually modify such data. Perl is another such example.